tcms/testruns/static/testruns/js/get.js

```diff
@@ -220,8 +220,8 @@ function renderAdditionalInformation(testExecutions, testExecutionCaseIds) {
 function renderTestExecutionRow(template, testExecution, testExecutionStatus) {
     template.find('.test-execution-checkbox').data('test-execution-id', testExecution.id)
     template.find('.test-execution-checkbox').data('test-execution-case-id', testExecution.case_id)
-    template.find('.list-group-item').addClass(`test-execution-${testExecution.id}`)
-    template.find('.list-group-item').addClass(`test-execution-case-${testExecution.case_id}`)
+    template.find('div.list-group-item').addClass(`test-execution-${testExecution.id}`)
+    template.find('div.list-group-item').addClass(`test-execution-case-${testExecution.case_id}`)
```

Comment:
You can pair that with a selector class like js-testexecution-row for each row and add it to the selector. Then it becomes much more explicit.